### PR TITLE
refactor: rename from go-ipfs to kubo

### LINKED ISSUE #51: Rename NPM package to kubo
> Part of https://github.com/ipfs/go-ipfs/issues/8959

## TODO

- [x] see if we can get `kubo` on NPM – tracked in  https://github.com/queerviolet/werkit/issues/1 (if it does not work out, we could go with `kubo-bin`)
- [x] decide if we want to do dual-publishing to both  names, or deprecate old one
- [ ] update CI to publish to new name

### PR DESCRIPTION
**Pull request by lidel**
Closes #51

### CODE REVIEW

File: .github/actions/check-for-go-ipfs-release/Dockerfile
Comment (galargh) [inline .github/actions/check-for-go-ipfs-release/Dockerfile:1]: We have to rename this action too.

File: .github/workflows/main.yml
[the remark this answers is not in the document]
Comment (galargh) [reply .github/workflows/main.yml:1]: I accepted the invite on behalf of ipfs service account which should resolve this.

**Comment by lidel** [comment]
Triggered manual build: https://github.com/ipfs/npm-kubo/actions/runs/5904170330/job/16015683148

File: src/download.js [hunk 1]
```diff
@@ -33,7 +33,7 @@ const isWin = process.platform === 'win32'
  * @param {string} url
  */
 async function cachingFetchAndVerify (url) {
-  const cacheDir = process.env.NPM_GO_IPFS_CACHE || cachedir('npm-go-ipfs')
+  const cacheDir = process.env.NPM_GO_IPFS_CACHE || cachedir('npm-kubo')
```
Comment (lidel) [inline src/download.js:36]: :information_source: missed env here, fixed in #65

**Pull request merged**